src/core_atmosphere/physics/physics_noaa/SMOKE/module_add_emiss_burn.F90

```diff
@@ -0,0 +1,181 @@
+!>\file  module_add_emiss_burn.F90
+!! This file adds the biomass burning emissions to the smoke field.
+
+module module_add_emiss_burn
+!RAR: significantly modified for the new BB emissions
+  use mpas_kind_types
+  use mpas_smoke_config
+CONTAINS
+  subroutine add_emis_burn(dtstep,dz8w,rho_phy,pi,ebb_min,          &
+                           !smoke,smoke_coarse,julday,gmt,xlat,xlong,     &
```

Comment:
is there a reason to keep these commented lines?

Reply:
Right now we have two versions. One is the experimental MPAS-Aerosol (Chemistry) with more chemical species (smoke_fine from wildfire, smoke_coarse from residential wood burning, dust, pollens, etc..). Another one is just smoke and dust for RRFSv2, which is planed in our ufs-comummuity/MPAS-Model authoritative repository. We want to share the same aerosol repository for the two versions, and also the similar smoke driver, which can easily add aerosol/chemical species. 
Thus we keep these commented lines for the two versions (MPAS-Aerosol and MPAS-Smoke/Dust).